models/yang/sonic/sonic-routing-policy-sets.yang

```diff
@@ -0,0 +1,175 @@
+module sonic-routing-policy-sets {
+    namespace "http://github.com/Azure/sonic-routing-policy-lists";
+    prefix srpolsets;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC ROUTING POLICY SETS";
+
+    revision 2019-10-23 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-routing-policy-sets {
+
+        container PREFIX_SET {
+
+            list PREFIX_SET_LIST {
+                key "name";
+
+                leaf name {
+                    type string {
+                        pattern '[a-zA-Z0-9]{1}([-a-zA-Z0-9_]{0,71})';
+                        length 1..72;
+                    }
+                }
+
+                leaf mode {
+                    type enumeration {
+                        enum IPv4;
+                        enum IPv6;
+                    }
+                }
+            }
+        }
+
+        container PREFIX {
+
+            list PREFIX_LIST { 
+                key "set_name ip_prefix masklength_range";
+
+                leaf set_name {
+                    mandatory true;
```

Comment:
Key is always mandatory, please remove "mandatory true". Same for all keys.

Reply:
removed